rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/90_unsupported_operations.yml

```diff
@@ -232,3 +232,91 @@ aggregate on _id:
             id:
               terms:
                 field: _id
+
+---
+synthetic source flattened field:
+  - skip:
+      version: " - 8.6.99"
+      reason: "synthetic source introduced in 8.7.0"
+
+  - do:
+      catch: /field \[k8s.pod.labels\] of type \[flattened\] doesn't support synthetic source/
+      indices.create:
+        index: test-flattened
+        body:
+          settings:
+            number_of_shards: 1
+            number_of_replicas: 0
+            index:
+              mode: time_series
+              routing_path: [ metricset, k8s.pod.uid ]
+              time_series:
+                start_time: 2021-04-28T00:00:00Z
+                end_time: 2021-04-29T00:00:00Z
+          mappings:
+            properties:
+              "@timestamp":
+                type: date
+              metricset:
+                type: keyword
+                time_series_dimension: true
+              k8s:
+                properties:
+                  pod:
+                    properties:
+                      uid:
+                        type: keyword
+                        time_series_dimension: true
+                      labels:
+                        type: flattened
+                      value:
+                        type: long
+                        time_series_metric: gauge
+
+---
+synthetic source text field:
+  - skip:
+      version: " - 8.6.99"
+      reason: "synthetic source introduced in 8.7.0"
+
+  - do:
+      catch: /field \[k8s.pod.agent.name\] of type \[text\] doesn't support synthetic source unless it is stored or has a sub-field of type \[keyword\] with doc values or stored and without a normalizer/
+      indices.create:
```

Comment:
I believe that not supporting `text` fields is a bigger problem than not supporting `flattened fields`. We should be able to support `text` fields as labels